### PR TITLE
OCM-9775 | test: Automate OCP-54469 OCP-75246

### PR DESCRIPTION
**Pull request by jameszwang**

### CODE REVIEW

File: tests/e2e/test_rosacli_account_roles.go [hunk 1]
```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
...
 		})
 })
+
+var _ = Describe("Create account roles", labels.Feature.AccountRoles, func() {
+	defer GinkgoRecover()
+
+	var (
+		rosaClient                     *rosacli.Client
+		ocmResourceService             rosacli.OCMResourceService
+		accountRolePrefixesNeedCleanup = make([]string, 0)
```
Comment (radtriste) [inline tests/e2e/test_rosacli_account_roles.go:787]: You do not need to create an empty array. It should be correctly handled by `len` and `append` methods
```suggestion
		accountRolePrefixesNeedCleanup []string
```

File: tests/e2e/test_rosacli_account_roles.go [hunk 2]
```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
...
+			Expect(output.String()).To(ContainSubstring("Error getting version: could not find versions for the " +
+				"provided channel-group: 'fakecg'"))
+
+		})
+
+	It("to create/Upgrade account-roles by setting version and channel-group via rosacli - [id:54469]",
+		labels.High, labels.Runtime.OCMResources,
+		func() {
+			defer func() {
+				By("Cleanup created account-roles in the test case")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_account_roles.go:839]: Shouldn't the cleanup be done into the OCM Resource Service directly ?

File: tests/e2e/test_rosacli_account_roles.go [hunk 3]
```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
...
+		accountRolePrefixesNeedCleanup = make([]string, 0)
+		path                           = "/aa/bb/"
+	)
+	BeforeEach(func() {
+		By("Init the client")
+		rosaClient = rosacli.NewClient()
+		ocmResourceService = rosaClient.OCMResource
+	})
+
+	It("to create account-roles with invalid version/channel group - [id:75246]",
```
Comment (radtriste) [inline tests/e2e/test_rosacli_account_roles.go:796]: What is the Runtime label ?

File: tests/e2e/test_rosacli_account_roles.go [hunk 4]
```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
...
+			Expect(err).To(BeNil())
+			Expect(versionC).NotTo(BeNil())
+			_, _, versionCandidate, err := versionC.MajorMinor()
+			Expect(err).To(BeNil())
+
+			channelVersion := map[string]string{"stable": versionStable, "candidate": versionCandidate}
+
+			By("Create/upgrade account role with version and channel group in auto mode")
+			for c, v := range channelVersion {
+				By("Create account roles in auto mode")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_account_roles.go:883]: Can you add the channel group to the `By` titles ? That way, it is easier to follow

File: tests/e2e/test_rosacli_account_roles.go [hunk 5]
```diff
@@ -777,3 +777,221 @@ var _ = Describe("List account roles", labels.Feature.AccountRoles, func() {
...
+					accrolePrefix, upgradeVersion))
+				Expect(output.String()).To(ContainSubstring("policy/%s-Support-Role-Policy' to version '%s'",
+					accrolePrefix, upgradeVersion))
+				Expect(output.String()).To(ContainSubstring("policy/%s-Installer-Role-Policy' to version '%s'",
+					accrolePrefix, upgradeVersion))
+			}
+
+			By("Create/upgrade account role with version and channel group in manual mode")
+			for c, v := range channelVersion {
+				By("Create account roles in manual mode")
```
Comment (radtriste) [inline tests/e2e/test_rosacli_account_roles.go:931]: Same here to add the channel group

File: tests/e2e/test_rosacli_account_roles.go (hunk 1 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_account_roles.go]: update according to comment.

File: tests/e2e/test_rosacli_account_roles.go (hunk 3 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_account_roles.go:811]: add Runtime label.

File: tests/e2e/test_rosacli_account_roles.go (hunk 4 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_account_roles.go]: add channel group to By titles.

File: tests/e2e/test_rosacli_account_roles.go (hunk 5 above)
Comment (jameszwang) [reply tests/e2e/test_rosacli_account_roles.go]: add channel group to By title.

**Pull request merged**